static/_redirects

```diff
@@ -30,6 +30,8 @@
 /docs/api-reference/labels-annotations-taints/     /docs/reference/labels-annotations-taints/ 301
 /docs/reference/using-api/api-overview/     /docs/reference/using-api/ 301
 /docs/reference/access-authn-authz/controlling-access/   /docs/concepts/security/controlling-access/ 301
+/docs/reference/kubectl/cheatsheet/   /docs/reference/kubectl/quick-guide/ 301
+/kubectlgudie /docs/reference/kubectl/quick-guide/ 301
```

Comment:
There's a minor typo, and the HTTP status should be 302 since this will be redirection, not a relocation.
```suggestion
/kubectlguide /docs/reference/kubectl/quick-guide/ 302
```

Reply:
i added the suggestion.... Thanks for review @dipesh-rawat